components/network-tools/src/test/java/org/datacleaner/extension/networktools/ResolveHostnameTransformerTest.java

```diff
@@ -19,38 +19,48 @@
  */
 package org.datacleaner.extension.networktools;
 
+import static org.junit.Assert.assertEquals;
+
 import org.datacleaner.data.MockInputColumn;
 import org.datacleaner.data.MockInputRow;
+import org.datacleaner.test.TestHelper;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ResolveHostnameTransformerTest {
+
+    private final ResolveHostnameTransformer t = new ResolveHostnameTransformer();
+    private final MockInputColumn<String> col = new MockInputColumn<String>("host", String.class);
+
+    @Before
+    public void setUp() {
+        t.hostnameColumn = col;
+    }
+
+    @Test
+    public void testTransformLocal() throws Exception {
+        assertEquals("127.0.0.1", t.transform(new MockInputRow().put(col, "localhost"))[0]);
+
+        assertEquals("127.0.0.1", t.transform(new MockInputRow().put(col, "127.0.0.1"))[0]);
+        
+        assertEquals(null, t.transform(new MockInputRow().put(col, ""))[0]);
+        
+        assertEquals(null, t.transform(new MockInputRow().put(col, null))[0]);
+        
+        assertEquals(null, t.transform(new MockInputRow().put(col,
+                "lmdslfsm flskmf lskmfls kmslf kdmlfsk mflsk fmsl kfdmsl"))[0]);
+    }
+    
+    @Test
+    public void testTransformInternet() throws Exception {
+        if (!TestHelper.isInternetConnected()) {
+            return;
```

Comment:
This is wrong-ish. You should use junit.Assume.assumeTrue(TestHelper.isInternetConnected()) instead.

Then the test gets properly skipped instead of claiming that the test passed, which we now nothing about.